msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -213,26 +215,31 @@ function onClickAddToPlaylist(video: IVideo) {
-| ------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------- |
+| Live Object    | Description                                                                                                                             |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
 | `LivePresence` | See which users are online, set custom properties for each user, and broadcast changes to their presence.                               |
-| `LiveEvent` | Broadcast individual events with any custom data attributes in the payload.                                                             |
-| `LiveState` | Similar to SharedMap, a distributed key-value store that allows for restricted state changes based on role, for example, the presenter. |
-| `LiveTimer` | Synchronize a countdown timer for a given interval.                                                                                     |
+| `LiveEvent`    | Broadcast individual events with any custom data attributes in the payload.                                                             |
+| `LiveState`    | Similar to SharedMap, a distributed key-value store that allows for restricted state changes based on role, for example, the presenter. |
+| `LiveTimer`    | Synchronize a countdown timer for a given interval.                                                                                     |
 
 ### LivePresence example
 
 :::image type="content" source="../assets/images/teams-live-share/live-share-presence.png" alt-text="Screenshot shows an example of showing people who available in a sessionTeams using Live Share presence.":::
 
-The `LivePresence` class makes tracking who is in the session easier than ever. When calling the `.initialize()` or `.updatePresence()` methods, you can assign custom metadata for that user, such as name or profile picture. By listening to `presenceChanged` events, each client receives the latest `LivePresenceUser` object, collapsing all presence updates into a single record for each unique `userId`.
+The `LivePresence` class makes tracking who is in the session easier than ever. When calling the `.initialize()` or `.updatePresence()` methods, you can assign custom metadata for that user, such as name, profile picture, or the identifier for content they are viewing. By listening to `presenceChanged` events, each client receives the latest `LivePresenceUser` object, collapsing all presence updates into a single record for each unique `userId`.
+
+Here are a few examples in which `LivePresence` can be used in your application:
```

Comment:
```suggestion
You can use ‘LivePresence' in your application in the following scenarios:
```

Reply:
These are just examples of scenarios and not an exhaustive list. Your wording implies that the provided scenarios are the only possible scenarios.